### PR TITLE
Add Raid potions to timer plugin

### PR DESCRIPTION
**Pull request by Sethtroll**

### CODE REVIEW

**Comment by Kamielvf** [comment]
The revitalisation is just the raids version of super restore - doesn't provide any buff so it should not have any timer related to it.

File: runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java [hunk 1]
```diff
@@ -325,6 +334,17 @@ public void onChatMessage(ChatMessage event)
 		{
 			createGameTimer(SANFEW);
 		}
+
+		if (event.getMessage().startsWith("You drink some of your") && event.getMessage().contains("overload") && client.getSetting(Varbits.IN_RAID) == 1 && config.showOverload())
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java]: As this message applies to both normal and raid overload, but you are specifically checking if player is in raids, at least the message check and config check should be merged to one if and then decide based on raid varbit.

File: runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java [hunk 2]
```diff
@@ -325,6 +334,17 @@ public void onChatMessage(ChatMessage event)
...
+		{
+			createGameTimer(OVERLOAD_RAID);
+		}
+
+		if (event.getMessage().startsWith("You drink some of your") && event.getMessage().contains("prayer enhance") && config.showPrayerEnhance())
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java]: Check config.show<myThing> before checking the message contents, as just checking for true/false is a lot faster than checking if string contains, starts or equals something.

File: runelite-api/src/main/java/net/runelite/api/Varbits.java [hunk 3]
```diff
@@ -180,7 +180,12 @@
 	/**
 	 * Motherlode mine sack
 	 */
-	SACK_NUMBER(5558, 375, 8, 15);
+	SACK_NUMBER(5558, 375, 8, 15),
+
+	/**
+	 * Raids
+	 */
+	IN_RAID(1431, 31, 31);
```
Comment (Adam-) [inline runelite-api/src/main/java/net/runelite/api/Varbits.java]: this is id 5432

**Comment by Kamielvf** [comment]
Potion timers are still up when you leave a raid with the boost active.

**Comment by deathbeam** [comment]
Yeah, probably hook for varbit changed needs to be added, and if raids one changed to 0, delete timers

**Pull request merged**